packages/story-editor/src/app/quickActions/useQuickActions.js

```diff
@@ -150,8 +151,21 @@ const useQuickActions = () => {
     [handleMouseDown]
   );
 
+  const hasAudioAnywhere = useStory(
+    ({ state }) =>
+      state.story.backgroundAudio ||
```

Comment:
```suggestion
      state.story?.backgroundAudio ||
```

This should mitigate some unrelated test failures